kolibri/plugins/learn/assets/src/actions.js

```diff
@@ -147,6 +197,42 @@ function redirectToLearnChannel(store) {
     });
 }
 
+function showExploreChannel(store, channelId) {
+  store.dispatch('CORE_SET_PAGE_LOADING', true);
+  store.dispatch('SET_PAGE_NAME', PageNames.EXPLORE_CHANNEL);
+  store.dispatch('SET_CURRENT_CHANNEL', channelId);
+  cookiejs.set('currentChannel', channelId);
+  ContentNodeResource.setChannel(channelId);
+
+  _getCurrentChannelRootTopicId()
+    .then((rootTopicId) => {
+      store.dispatch('SET_ROOT_TOPIC_ID', rootTopicId);
+      const attributesPromise = ContentNodeResource.getModel(rootTopicId).fetch();
+      const childrenPromise = ContentNodeResource.getCollection({ parent: rootTopicId }).fetch();
+      const channelPromise = ChannelResource.getCollection({}).fetch();
+      Promise.all([attributesPromise, childrenPromise, channelPromise])
+        .then(([attributes, children, channelList]) => {
+          const pageState = { rootTopicId };
+          pageState.topic = _topicState(attributes);
+          const collection = _collectionState(children);
+          pageState.subtopics = collection.topics;
+          pageState.contents = collection.contents;
+          store.dispatch('SET_PAGE_STATE', pageState);
+          store.dispatch('CORE_SET_PAGE_LOADING', false);
+          store.dispatch('CORE_SET_ERROR', null);
+          store.dispatch('SET_CHANNEL_LIST', channelList);
+        })
+        .catch((error) => {
+          store.dispatch('CORE_SET_ERROR', JSON.stringify(error, null, '\t'));
+          store.dispatch('CORE_SET_PAGE_LOADING', false);
+        });
+    })
+    .catch((error) => {
+
+    });
```

Comment:
If we're not doing anything with the error, we _probably_ shouldn't have this block